rpc/src/rpc_pubsub_service.rs

```diff
@@ -193,7 +193,7 @@ pub struct TestBroadcastReceiver {
 #[cfg(test)]
 impl TestBroadcastReceiver {
     pub fn recv(&mut self) -> String {
-        match self.recv_timeout(std::time::Duration::from_secs(10)) {
+        match self.recv_timeout(std::time::Duration::from_secs(50)) {
```

Comment:
I looked at test_check_program_subscribe_for_missing_optimistically_confirmed_slot_with_no_banks_no_notifications -- it is expecting recv to panic -- probably expecting it to timeout. Increasing timeout will simply make the test run longer -- but it might still panic. We should review the test code, is it legit to expect this panic and timeout? If so, probably we should reduce the timeout instead of increasing it to make it run faster. Otherwise, we should make the test code run faster not waiting for 50 seconds. Any scheduling cannot account for 10 seconds in my opinion.

Reply:
// The following should panic
        let _response = receiver.recv();

Reply:
The test code simply makes tokio calls.  There is not much it can do to run faster because unless there is major design change to move away from using tokio.

Reply:
I debugged rpc code timeout issues before, and found that there is no real-time control on the timing.  Most runs pass quickly, but there are always long-tail runs which far exceeding the given timeouts.  This is the problem which has causing many failures in the regression tests, forcing everyone to retry many times when trying to commit new code.